pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/stoolupload/StoolUploadDao.java

```diff
@@ -0,0 +1,95 @@
+package org.broadinstitute.dsm.db.dao.stoolupload;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Optional;
+
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.dsm.db.dao.Dao;
+import org.broadinstitute.dsm.db.dto.stoolupload.StoolUploadDto;
+import org.broadinstitute.dsm.model.KitDDPNotification;
+import org.broadinstitute.dsm.statics.ApplicationConfigConstants;
+import org.broadinstitute.dsm.statics.DBConstants;
+import org.broadinstitute.dsm.util.DSMConfig;
+import org.broadinstitute.dsm.util.EventUtil;
+import org.broadinstitute.lddp.db.SimpleResult;
+
+public class StoolUploadDao implements Dao<StoolUploadDto> {
+    private static final String SELECT_KIT_ID = "SELECT dsm_kit_id FROM ddp_kit INNER JOIN ddp_kit_request dkr"
+            + " on ddp_kit.dsm_kit_request_id = dkr.dsm_kit_request_id";
+    private static final String BY_BARCODE = " WHERE kit_label = ?";
+    private static final String BY_PT_ID = " dkr.ddp_participant_id = ?";
+    private static final String UPDATE_KIT = "UPDATE ddp_kit SET receive_date = ?, receive_by = 'HSPH'";
+    private static final String BY_KIT_ID = " WHERE dsm_kit_id = ?";
+    private static final String IF_NULL = " AND receive_date IS NULL";
```

Comment:
same here. no need to split them if there are only used here and only ones. 
I would recommend to only split them if we need to reuse them